src/datapump/clients/data_api.py

```diff
@@ -101,6 +103,7 @@ def create_vector_version(
         indices: List[Dict[str, Any]],
         cluster: Dict[str, Any],
         table_schema: List[Dict[str, Any]],
+        constraints: List[Dict[str, Any]],
```

Comment:
Since you added a new argument `constraints` to `create_vector_version()`, you need to fix the call at line 165 below, which is not using keyword arguments.  (Github is not letting me add a comment on that line.)

You also have a call to `create_vector_version` at geotrellis.py:202 that you didn't change, but it uses keywords for the later arguments.  But is it OK that constraints has a value of None, since it is not passed in (seems likely, but might want to put in the arg explicitly anyway)?

Reply:
good catch, thanks!

Reply:
Wow, that's a thorough review! I agree, the call in copy_version looks like it needs to change.